### ISSUE TITLE
Add Embedding Flag info to fonts setting display and typography page

### ISSUE THREAD

**Issue by ronssssss** [MicrosoftDocs/typography-issues#892]
page says <snip> Font files contain flags that indicate if and how they can be embedded within a document file. </snip>
great. Where does the average user find these flags. They are not displayed in the Fonts dialog. They are not displayed on the Typography info page: https://docs.microsoft.com/en-us/typography/font-list/ink-free 
.
This information should be easily available in the fonts setting.  It would be nice on the typography page too.
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1ea5ad39-ec78-af12-1479-e3def2c62723
* Version Independent ID: 4c9dbe2a-0d30-42fe-34be-38053fcf5c4e
* Content: [Font redistribution FAQ - Typography](https://docs.microsoft.com/en-us/typography/fonts/font-faq)
* Content Source: [typographydocs/fonts/font-faq.md](https://github.com/MicrosoftDocs/typography/blob/live/typographydocs/fonts/font-faq.md)
* Product: **typography**
* GitHub Login: @alib-ms
* Microsoft Alias: **alib**

**Comment by PeterCon** [comment]
To suggest a feature in Windows, please make the suggestion in the Windows Feedback Hub app on Windows 10 / Windows 11.

**Comment by alib-ms** [comment]
I agree with you "This information should be easily available in the fonts setting." I will communicate this with the Fonts Settings team. Meanwhile, the good news, there is a workaround
Click the Search icon on the taskbar. Type "Fonts" and select Control Panel > Fonts. 
Click on one of the fonts and you will see the embedding information as shown below.  
![image](https://user-images.githubusercontent.com/36778811/190692835-8b799988-70ee-4ad9-8144-36f8deab5a5d.png)

**Comment by kaanchan** [comment]
@alib-ms That was a very useful suggestion for viewing the font embeddability setting. Thank you. I was experiencing the same as @ronssssss
